JAGPropertyConverter/JAGPropertyConverter.m

```diff
@@ -638,4 +661,13 @@ - (BOOL)_isKeyPathKey:(NSString *)key {
     return [key rangeOfString:@"."].location != NSNotFound;
 }
 
+// clear property value (set property to nil or 0 if it is primitive data type)
+- (void)_clearObjectValue:(id)object forProperty:(JAGProperty *)property {
```

Comment:
i thought about it, but was too lazy to add ... but yeah, should have added already in the first place :D